admin/easypopulate_4.php

```diff
@@ -213,20 +213,20 @@
 /* @var $epdlanguage_query type array */
 //$epdlanguage_query = $db->Execute("SELECT languages_id, name FROM ".TABLE_LANGUAGES." WHERE code = '".DEFAULT_LANGUAGE."'");
 if (!defined(DEFAULT_LANGUAGE)) {
-  $epdlanguage_query = ep_4_query("SELECT languages_id, code FROM " . TABLE_LANGUAGES . " ORDER BY languages_id LIMIT 1");
```

Comment:
This style of query offers in house error management and does not crash to resolve the issue by use of the ZC /cache or /logs directory or some other "outside" location, but instead provide a debug file that is readily available "on screen" for review.